native/cocos/base/threading/MessageQueue.cpp

```diff
@@ -22,9 +22,15 @@
  THE SOFTWARE.
 ****************************************************************************/
 
+#include <unistd.h>
```

Comment:
MessageQueue is used for all platforms, and this header file is not valid for all platforms.

Reply:
Thanks, made the adjustment in 23d0327